s/features/notes/models/cache/notes-cache.ts

```diff
@@ -142,11 +142,18 @@ export function prepareNotesCacheCreator({
 
 			async markSpecificNoteOld(noteId: string) {
 				await markNotesNewOrOld(true, [noteId])
+				cacheState.readable.pageNumber > totalNumberOfPages()
+				? cacheState.writable.pageNumber = 1
```

Comment:
indentation here is wrong.

- the ternary `?` and `:` parts should be indented one level compared to the ternary condition